### PR TITLE
Moved to Failed errors framework

### PR DESCRIPTION
**Pull request by Sumrachek**

### CODE REVIEW

File: ton_sdk/Cargo.toml [hunk 1]
```diff
@@ -3,11 +3,11 @@ name = "ton_sdk"
 version = "0.1.0"
 
 [dependencies]
-ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos" }
+ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos-2" } # Fix branch before merge!!!
 ton_block = { git = "ssh://git@github.com/tonlabs/ton-block.git" }
 ton_vm = { git = "ssh://git@github.com/tonlabs/ton-vm.git", default-features = false }
 ton_types = { git = "ssh://git@github.com/tonlabs/ton-types.git" }
-ton_executor = { git = "ssh://git@github.com/tonlabs/ton-executor.git" }
+ton_executor = { git = "ssh://git@github.com/tonlabs/ton-executor.git", branch = "feuture-failure-errors" } # Fix branch before merge!!!
```
Comment (AlexeyVavilin) [inline ton_sdk/Cargo.toml]: `error-chain` dependency should be deleted I think

File: ton_sdk/Cargo.toml [hunk 2]
```diff
@@ -3,11 +3,11 @@ name = "ton_sdk"
 version = "0.1.0"
 
 [dependencies]
-ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos" }
+ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos-2" } # Fix branch before merge!!!
```
Comment (AlexeyVavilin) [inline ton_sdk/Cargo.toml]: Fix branches. `ton_abi` and `ton_executor` are merged

File: ton_sdk/Cargo.toml (hunk 1 above)
Comment (Sumrachek) [reply ton_sdk/Cargo.toml]: Done

File: ton_sdk/Cargo.toml (hunk 2 above)
Comment (Sumrachek) [reply ton_sdk/Cargo.toml]: Done

**Comment by AlexeyVavilin** [comment]
Create branch `0.20.0-rc` from `master` and make PR into them

**Pull request closed without merging**